violet-framework/src/main/java/com/horstmann/violet/workspace/editorpart/behavior/AddNodeBehavior.java

```diff
@@ -91,15 +91,16 @@ public void onMouseClicked(MouseEvent event)
 
     private boolean checkIfNumberOfInitialNodesIsMoreThanOne(INode newNode)
     {
+        boolean result = false;
         if (newNode instanceof IInitialStateNode)
         {
             int numberOfInitialNodes = getNumberOfInitialNodes();
```

Comment:
do you really need this variable? I can see, that you use it only in line 98. 
furthermore - it's ok :)